code/modules/crafting/recipes.dm

```diff
@@ -555,7 +555,10 @@
 	name = "Pressure Plate"
 	result = /obj/item/device/pressure_plate
 	time = 5
-	reqs = list(/obj/item/stack/sheet/plasteel = 1, /obj/item/stack/tile/plasteel = 1, /obj/item/stack/cable_coil = 2)
+	reqs = list(/obj/item/stack/sheet/metal = 1,
+				  /obj/item/stack/tile/plasteel = 1,
```

Comment:
you stupit rabbit its still here

Reply:
That's a tile dummy

Reply:
You can't even make plasteel floortiles

Dumb Kor doesn't even play!

Reply:
true